cmd/snap-update-ns/change_test.go

```diff
@@ -749,14 +748,6 @@ func (s *changeSuite) TestNeededChangesParallelInstancesInsideMountNew(c *C) {
 	})
 }
 
-func mustReadProfile(profileStr string) *osutil.MountProfile {
```

Comment:
Looks like 851c63a985d52661b834d2f49c2d089c66691609 removed any references to this code.